package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1098,8 +1098,19 @@ def omega_selections(residues, c_name='C', n_name='N', ca_name='CA'):
     transplants[ResidueGroup].append(('omega_selections', omega_selections))
 
     def chi1_selection(residue, n_name='N', ca_name='CA', cb_name='CB',
-                       cg_name='CG'):
-        """Select AtomGroup corresponding to the chi1 sidechain dihedral N-CA-CB-CG.
+                       cg_name='CG CG1 OG OG1 SG'):
+        r"""Select AtomGroup corresponding to the chi1 sidechain dihedral ``N-CA-CB-*G.``
+        The gamma atom is taken to be the heavy atom in the gamma position. If more than one
+        heavy atom is present (e.g. CG1 and CG2), the one with the lower number is used (CG1).
+
+        .. warning::
+
+            This numbering of chi1 atoms here in following with the IUPAC 1970 rules\.
```

Comment:
Is there a reason for the backslash escaped period, @lilyminium ?

I would have changed it but maybe there's something I am missing.

Reply:
Accident, sorry.